src/java/org/apache/cassandra/db/CounterMutationVerbHandler.java

```diff
@@ -48,7 +66,44 @@ public void doVerb(final Message<CounterMutation> message)
         // it's own in that case.
         StorageProxy.applyCounterMutationOnLeader(cm,
                                                   localDataCenter,
-                                                  () -> MessagingService.instance().send(message.emptyResponse(), message.from()),
+                                                  () -> respond(message, message.from()),
                                                   queryStartNanoTime);
     }
+
+    private void respond(Message<?> respondTo, InetAddressAndPort respondToAddress)
+    {
+        Message.Builder<NoPayload> response = respondTo.emptyResponseBuilder();
+        addSensorsToResponse(response);
+
+        MessagingService.instance().send(response.build(), respondToAddress);
+    }
+
+    private void addSensorsToResponse(Message.Builder<NoPayload> response) {
+        // Add write bytes sensors to the response
+        Function<String, String> requestParam = SensorsCustomParams::encodeTableInWriteByteRequestParam;
+        Function<String, String> tableParam = SensorsCustomParams::encodeTableInWriteByteTableParam;
+        RequestSensors requestSensors = RequestTracker.instance.get();
```

Comment:
This will actually not work (in the sense `requestSensors` will be null) because the response is handled in another thread in the `RequestResponseStage` - this is by design to as per this [comment](https://github.com/datastax/cassandra/blob/37aac13fed4237805e3942ffbdc54f68eef7b96f/src/java/org/apache/cassandra/service/StorageProxy.java#L356-L361). It works fine when the counter mutation is directly applied on the coordinator (which is not the case in cndb)

Reply:
Fixed [here](https://github.com/datastax/cassandra/pull/1056/commits/5860bcb704d5df87240d452a3a90788d10f0553e) along side proper collection of mutation written bytes as reported by replicas